depot/io/local.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, file_id, local_path):
         except:
             raise IOError('File %s not existing' % file_id)
 
-        metadata_info = {'filename': 'unknown',
+        metadata_info = {'filename': 'unnamed',
```

Comment:
Shouldn't we asume that filename and content_type are always in metadata? Why repeat here default values? 
Is it for backwards compatibility?